app/src/highlighter/globals.d.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable typescript/interface-name-prefix */
+/* eslint-disable @typescript-eslint/interface-name-prefix */
```

Comment:
What do you think about getting rid of this rule? Seems like the community has moved away from the `I` prefix.

Reply:
I've grumbled about this before, but I'd prefer to discuss this outside the scope of this PR and review our use of interfaces overall. I'll open up an issue to braindump my thoughts about this later today.

Reply:
#6773